components/eamxx/scripts/scripts-tests

```diff
@@ -667,12 +592,6 @@ OR
     parser.add_argument("-m", "--machine",
                         help="Provide machine name. This is required for full (not dry) runs")
 
-    parser.add_argument("-g", "--generate", action="store_true",
```

Comment:
Does it make sense to keep `-f` now that `-g` and `-c` have been removed? Seems like we do run full regardless now, no? I mean, what's the point of running without `-f`?

Reply:
We still need to distinguish between "full" and "jenkins" runs.

Reply:
Actually, we could just assume no `-j` implies `-f`. Let me look

Reply:
There's still some value here since, without `-f` or `-j`, it just runs very quick pylint and doctests, which is useful.